docs2/site/docs/getting-started/variables.md

```diff
@@ -36,6 +36,6 @@ var inputs = variablesJson.ToInputs();
 await schema.ExecuteAsync(_ =>
 {
   _.Query = "...";
-  _.Inputs = inputs;
+  _.Variables = inputs;
```

Comment:
I did a quick search for changes needed to the documentation.  Very likely I still missed something.